examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/main/offerings/OfferingsScreen.kt

```diff
@@ -129,6 +174,59 @@ private fun OfferingsListScreen(
                 .build(),
         )
     }
+
+    if (showDialog.value) {
+        Dialog(
+            onDismissRequest = {
+                showDialog.value = false
+                noPlacementFoundMessage.value = null
+                placementIdentifier.value = ""
+            },
+            properties = DialogProperties(usePlatformDefaultWidth = false),
```

Comment:
Hmm is this needed? This is usually used if you want to make the dialog full screen, and not sure if we want to do that for this one?

Reply:
This actually is 🤷‍♂️  It resizes the model when placement is not found so things refit 👇 

<img width="352" alt="Screenshot 2024-02-20 at 12 20 09 PM" src="https://github.com/RevenueCat/purchases-android/assets/401294/2a28d39a-932f-4bb9-89b3-51614635615a">

BTW, I'm _really_ bad at jetpack compose so any suggestions are appreciated 😁